kbfsgit/runner.go

```diff
@@ -747,6 +948,14 @@ func (r *runner) handleOption(ctx context.Context, args []string) (err error) {
 		r.progress = b
 		r.log.CDebugf(ctx, "Setting progress to %t", b)
 		result = "ok"
+	case gitOptionCloning:
```

Comment:
I don't see a cloning related option defined here: https://git-scm.com/docs/git-remote-helpers#_options Is it something`git` passes in to the helper automatically during a clone, or something user needs to provide?

If former, how come the cloning already worked before this is added now? Is it a phase that `git` simply skips if it gets "unsupported" line from helper?

Reply:
You don't see that option because you're reading the docs for git 1.8.1.4.  Here are the latest docs: https://git-scm.com/docs/gitremote-helpers#gitremote-helpers-emoptioncloningememtrueememfalseem

It's just a hint `git` gives to the remote helper.  `git` itself always just sends `fetch` commands, and it's up to the helper to decide whether it wants to optimize for cloning or not.

Reply:
Oh I see; thanks for the link!